src/components/reusables/ActionBar/ActionBar.test.cy.tsx

```diff
@@ -0,0 +1,153 @@
+  });
+
+  it('non expandable version should not render chevron when it is rendered ', () => {
+    cy.mount(nonExpandableActionBar());
+    cy.get('[data-testid="action-bar-icon"]').should('not.exist');
+  });
+
+  it('should not render chevron when non expandable versjon is rendered ', () => {
+    cy.mount(nonExpandableActionBar());
+    cy.get('[data-testid="action-bar-icon"]').should('not.exist');
+  });
+
+  it('should call handleClick when expandable ActionBar is clicked by mouse', () => {
+    const handleClick = () => {
+      cy.stub();
+    };
+
+    const handleClickSpy = cy.spy(handleClick).as('handleClickSpy');
+
+    cy.mount(expandableActionBar({ onClick: handleClickSpy }));
+
+    cy.get('[data-testid="action-bar"]').click();
+    cy.get('@handleClickSpy').should('have.been.called');
+  });
+
+  it('should call handleClick when expandable ActionBar is clicked by space', () => {
+    const handleClick = () => {
+      cy.stub();
+    };
+    const handleClickSpy = cy.spy(handleClick).as('handleClickSpy');
+
+    cy.mount(expandableActionBar({ onClick: handleClickSpy }));
+
+    cy.get('[data-testid="action-bar"]').type(' ');
+    cy.get('@handleClickSpy').should('have.been.called');
+  });
+
+  it('should call handleClick when expandable ActionBar is clicked by enter', () => {
+    const handleClick = () => {
+      cy.stub();
+    };
+    const handleClickSpy = cy.spy(handleClick).as('handleClickSpy');
+
+    cy.mount(expandableActionBar({ onClick: handleClickSpy }));
+
+    cy.get('[data-testid="action-bar"]').type('{enter}');
+    cy.get('@handleClickSpy').should('have.been.called');
+  });
+
+  it('should not call handleClick when non expandable ActionBar is clicked', () => {
+    const handleClick = () => {
+      cy.stub();
+    };
+
+    const handleClickSpy = cy.spy(handleClick).as('handleClickSpy');
+
+    cy.mount(nonExpandableActionBar({ onClick: handleClickSpy }));
+
+    cy.get('[data-testid="action-bar"]').click();
+    cy.get('@handleClickSpy').should('have.been.called');
+  });
+
+  it('should only call handleActionClick when an action in ActionBar is clicked', () => {
+    const handleClick = () => {
+      cy.stub();
+    };
+
+    const handleClickSpy = cy.spy(handleClick).as('handleClickSpy');
+
+    const handleActionClick = () => {
+      cy.stub();
+    };
+
+    const handleActionClickSpy = cy.spy(handleActionClick).as('handleActionClickSpy');
+
+    const actionButton = <Button onClick={handleActionClickSpy}>Action</Button>;
+    cy.mount(nonExpandableActionBar({ actions: actionButton, onClick: handleClickSpy }));
+
+    cy.findByRole('button', { name: /Action/i }).click();
+    cy.get('@handleActionClickSpy').should('have.been.called');
+    cy.get('@handleClickSpy').should('not.have.been.called');
+  });
+
+  it('should have aria-expanded=false when open=false and not show content', () => {
+    cy.mount(expandableActionBar({ open: false }));
+    cy.get('div').should('not.contain', 'Content');
```

Comment:
It checks that ActionBarContent with "Content" in line 25 in this files isn't dsiplayed

Reply:
Ok, it looked like it was checking that there existed at least one div that didn't contain 'Content', but I tested locally and it seems it checks that no div contains 'content' so that makes sense.